apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Tests/LruCacheTest.cs

```diff
@@ -128,7 +128,7 @@ public void MultithreadedUse()
             }
             for (int i = 0; i < threadCount; i++)
             {
-                Assert.True(threads[i].Join(TimeSpan.FromSeconds(5)));
+                Assert.True(threads[i].Join(TimeSpan.FromSeconds(5).Milliseconds));
```

Comment:
This should be `.Join((int)TimeSpan.FromSeconds(5).TotalMilliseconds)`.
`.Milliseconds` returns just the millisecond part, which is 0. This was the cause of the test failure.